CONTRIBUTING.md

```diff
@@ -31,6 +31,17 @@ on the new feature or improvement.
 
 ## Code changes
 
+### IntelliJ IDEA tips
+
+Nessie uses antlr and protoc to generate source files. Those files are needed to build the Nessie project.
+
+Tip: In the "Project" view (usually on the left) right-click on the `[nessie]` project, open the
+`Maven` sub-menu (at the bottom of the context menu), click on `Generate Sources and Update Folders`.
+This will generate the protobuf source files that necessary to build Nessie via IntelliJ IDEA.
+
+Note: Using the above method via the project's context menu is much quicker than running
+`Generate Sources and Update Folders for all projects` from the Maven view (usually on the right).
```

Comment:
does the tip above mean to select a specific sub-project?
otherwise it is not immediately clear how "all projects" would be different from doing the same on the root project :sweat_smile: 
if its odd intellj behavior we should maybe mention that.

Reply:
`Generate Sources and Update Folders for all projects` is the name of the IntelliJ action

Reply:
i know, my point was that its not immediately (at least to me) why running the action on the root project on the left has different behavior than running it on "all projects" on the right (since intellij only has a single project open and all project means all maven module afaict). no big deal.